manifest.json

```diff
@@ -6,7 +6,7 @@
         "en": "Web interface for Youtube-dl",
         "fr": "Interface Web pour Youtube-dl"
     },
-    "version": "3.0.3~ynh1",
+    "version": "3.0.1~ynh1",
```

Comment:
typo on version ? 
shouldn't it be : `3.0.3~ynh2` ?

Reply:
yes, my bad, it was here because of my tests for the autoupgrade script